packages/devtools/lib/src/timeline/frames_bar_plotly.dart

```diff
@@ -42,6 +42,32 @@ class FramesBarPlotly {
   static const int ticksInRangeSlider = 90;
 
   final String _domName;
+  final bool useLogScale;
+
+  final yAxisLogScale = AxisLayout(
```

Comment:
nit: define types for top level fields. also make these private as they shouldn't be visible to users of the class.

Reply:
Done.